grails-app/services/org/pih/warehouse/core/UserService.groovy

```diff
@@ -51,14 +52,8 @@ class UserService {
         // to compare the password with confirm password before
         // setting the new password in the database
         userInstance.properties = params
-        if (params.changePassword && userInstance.password != params.password) {
-            userInstance.password = params?.password?.encodeAsPassword()
-            userInstance.passwordConfirm = params?.passwordConfirm?.encodeAsPassword()
-        } else {
-            // Needed to bypass the password == passwordConfirm validation
-            userInstance.passwordConfirm = userInstance.password
-        }
-
```

Comment:
have you made sure this method is not used for imports where we probably wouldn't want to remove this part?

Reply:
I was able to import users, in the import users template there is no password column
![image](https://github.com/openboxes/openboxes/assets/83239466/d3c4d0d9-109d-431b-9343-db23cbd9a289)